app/assets/javascripts/utilities/buildArticleHTML.js.erb

```diff
@@ -97,11 +97,8 @@ function buildArticleHTML(article) {
       }
     }
     var readingTimeHTML = '';
-    if (article.reading_time && article.class_name === "Article") {
-      readingTimeHTML = '<a href="'+article.path+'" class="article-reading-time">'+ (article.reading_time < 2 ? '1 min' : article.reading_time + ' min') +' read</a>'
-    } else if (article.class_name === "Article") {
-      readingTimeHTML = '<a href="'+article.path+'" class="article-reading-time">3 min read</a>'
-    }
+    readingTimeHTML = '<a href="'+article.path+'" class="article-reading-time">'+ (article.reading_time < 1 ? '1 min' : article.reading_time + ' min') +' read</a>'
```

Comment:
You still need the check `if article.class_name === "Article"`, unfortunately this monster function can receive arguments that aren't `Article` objects